src/app/api/mock/onerep/scan-results/route.ts

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { errorIfProduction } from "../../utils/errorThrower";
+import { MOCK_ONEREP_BROKERS } from "../config/config";
+import { NextRequest, NextResponse } from "next/server";
+
+export function GET(req: NextRequest) {
+  const prodError = errorIfProduction();
+  if (prodError) return prodError;
+
+  const page = req.url.match(/page=([0-9]+)/)![1] || "1";
```

Comment:
For HTTP params, there is a convenience method so you get a [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) (similar to a `Map`) interface (https://nextjs.org/docs/app/api-reference/functions/next-request#nexturl), for example:
```
req.nextUrl.searchParams.get("page")
```